apps/frontend/src/renderer/components/github-prs/components/PRFilesViewer.tsx

```
@@ -0,0 +1,317 @@
+
+  if (!pr.files.length) {
+    return null;
+  }
+
+  return (
+    <div className="space-y-2">
+      {/* Header with file toggle */}
+      <button
+        onClick={() => setShowFiles(!showFiles)}
+        className="text-sm font-medium flex items-center gap-2 hover:text-foreground text-foreground/80 transition-colors"
+      >
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)